src/annotate.rs

```diff
@@ -26,8 +32,6 @@ pub enum MemberId<'a> {
     Variant,
 }
 
-/// Trait implemented on structs to inform the serializer about formatting
-/// options and comments.
```

Comment:
Is there a reason for removing this doc comment?

Reply:
Nope, just a mistake.